blackbox/docs/admin/auth/hba.txt

```diff
@@ -160,3 +159,33 @@ the the ``auth.host_based`` setting, like this:
         config:
           0:
             user: crate
+
+Authenticating to Admin UI
+==========================
+
+.. hide:
+
+    cr> CREATE USER admin;
+    CREATE OK, 1 row affected (... sec)
+
+When trying to access the Admin UI of CrateDB, initially an authentication with
```

Comment:
`initially an authentication with..` sounds wrong to me. Same with `Authentication to Admin UI`  - did a native speaker take a look at this?

Reply:
Will wait for @nslater suggestion there.

Reply:
just "authentication" is better, no "an"